pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -120,14 +122,19 @@ func ReconcileDevWorkspace(deployContext *deploy.DeployContext) (bool, error) {
 		return true, nil
 	}
 
+	// do nothing if dev workspace is disabled
+	if !deployContext.CheCluster.Spec.DevWorkspace.Enable {
+		return true, nil
+	}
+
 	// Check if exists devworkspace operator csv is already installed
 	devWorkspaceOperatorCSVExists := isDevWorkspaceOperatorCSVExists(deployContext)
 	if devWorkspaceOperatorCSVExists {
 		return true, nil
 	}
 
-	// do nothing if dev workspace is disabled
-	if !deployContext.CheCluster.Spec.DevWorkspace.Enable {
+	if checkIfCheIsInstalledInSingleNamespacesMode(deployContext) && util.IsOpenShift {
+		logrus.Warnf(`DevWorkspace Operator can't be enabled because the operator was installed in Single Namespace mode.`)
```

Comment:
It is better to say something like this:
"Deploy Eclipse Che from <...> channel to deploy DevWorkspace operator."